src/app/clients/edit/[id]/autofill.ts

```diff
@@ -25,15 +15,22 @@ const autofill = (
     clientData: Schema["clients"],
     familyData: Schema["families"][]
 ): ClientFields => {
-    const children = familyData
-        .filter((member) => !isAdult(member))
-        .map((child): Person => {
-            return {
-                gender: child.gender,
-                age: child.age,
-                primaryKey: child.primary_key,
-            };
-        });
+    const children = familyData.filter(isChildPerson).map(
+        (child): Person => ({
+            gender: child.gender,
+            birthYear: child.birth_year,
+            birthMonth: child.birth_month,
+            primaryKey: child.primary_key,
+        })
+    );
+
+    const adults = familyData.filter(isAdultPerson).map(
+        (adult): Person => ({
+            gender: adult.gender,
+            birthYear: adult.birth_year,
+            primaryKey: adult.primary_key,
+        })
+    );
```

Comment:
Is it possible to use the same `getFormattedPeople` function here, same as in `formatFamiliesData.ts`? Or is there some difference that I can't see on first glance?

Reply:
Yes I think the primary key doesn't get used in the form so I can use the same function.